packages/web-components/fast-components/src/avatar/fixtures/base.html

```diff
@@ -0,0 +1,108 @@
+<fast-design-system-provider use-defaults>
+    <style>
+        fast-avatar {
+            --avatar-fill-accent-primary: #cf4073;
```

Comment:
Is this a different case than other token APIs because they involve custom names? With other components we've avoided prefixing them with the component name.